skrub/datasets/_utils.py

```diff
@@ -11,13 +41,14 @@ def get_data_dir(name: str | None = None) -> Path:
 
     Parameters
     ----------
+    data_home : Path, default=None
+        The path to skrub data directory. If `None`, the default path
+        is `~/skrub_data`.
     name: str, optional
         Subdirectory name. If omitted, the root data directory is returned.
```

Comment:
The parameter order must match the signature's :)
Also, by convention, `default=None` is replaced by `optional`

```suggestion
    name: str, optional
        Subdirectory name. If omitted, the root data directory is returned.
    data_home : pathlib.Path or str, optional
        The path to skrub data directory. If `None`, the default path
        is `~/skrub_data`.
```